manta-pay/abstract-protocol.tex

```diff
@@ -59,14 +88,60 @@ \subsection{Abstract Cryptographic Schemes}
     \end{itemize}
```

Comment:
Which choice are you referring to? Choosing secret keys? The secret keys should be chosen uniformly at random from the secret key domain. The shared secret is then generated from two randomly chosen secret keys. If a participant chooses a secret key in another way, we can't guarantee that it will be a good secret. Like, no one should choose `0` as a secret key.

Reply:
Either remove "No Key Control" or fix it to prevent that case. @bhgomes